cloud_utils/cache/file_store.py

```diff
@@ -83,9 +83,22 @@ def load_file_from_bucket(bucket_name: Text, file_name: Text):
     )
 
 
+def hash_exist_on_bucket(bucket_name: str, file_name: str) -> bool:
+    return gamla.pipe(
```

Comment:
i think the pipe here is an overkill just do:

`storage.blob_exists(bucket_name, utils.hash_to_filename(file_name))`